### PR TITLE
Shorten telegram callback data

### PR DESCRIPTION
**Pull request by Konstantinov-Innokentii**

### CODE REVIEW

File: engine/apps/telegram/tests/test_keyboard_renderer.py [hunk 1]
```diff
@@ -49,27 +49,27 @@ def test_actions_keyboard_alerting(make_organization, make_alert_receive_channel
         [
             InlineKeyboardButton(
                 text="Acknowledge",
-                callback_data=f"{alert_group.pk}:acknowledge:oncall-uuid{organization.uuid}",
+                callback_data=f"{alert_group.pk}:0:oncall-uuid{organization.uuid}",
```
Comment (joeyorlando) [inline engine/apps/telegram/tests/test_keyboard_renderer.py:52]: should these be pulled from `ACTION_TO_CODE_MAP`, rather than hardcoding them here?

Comment (Konstantinov-Innokentii) [reply engine/apps/telegram/tests/test_keyboard_renderer.py:52]: I hardcoded it's intentionally, to make tests more strict and be protected from changes in the ACTION_TO_CODE_MAP

Comment (joeyorlando) [reply engine/apps/telegram/tests/test_keyboard_renderer.py:52]: ohhh sorry.. this is tests, not the actual implementation, my bad

File: engine/apps/telegram/renderers/keyboard.py [hunk 2]
```diff
@@ -80,12 +99,14 @@ def unsilence_button(self) -> InlineKeyboardButton:
...
+        callback_data_args = [self.alert_group.pk, action_code]
         if action_data is not None:
             callback_data_args.append(action_data)
-        # Add org id with 'oncall-uuid' prefix to callback data.
-        # It's a workaroung to pass org_id to the oncall-gateway while proxying requests.
-        # TODO: switch to json str instead of ':' separated string.
+        # Add org id with 'oncall' prefix to callback data.
+        # It's a workaroung to pass oncall_-uuid to the oncall-gateway while proxying requests.
+        # TODO: check if it's possible switch to json str instead of ':' separated string.
+        # Note, that is't 64bytes limit to callback data
```
Comment (joeyorlando) [inline engine/apps/telegram/renderers/keyboard.py:109]:
```suggestion
        # Note, that there is a 64bytes limit to callback data
```

File: engine/apps/telegram/renderers/keyboard.py [hunk 3]
```diff
@@ -80,12 +99,14 @@ def unsilence_button(self) -> InlineKeyboardButton:
...
-        callback_data_args = [self.alert_group.pk, action.value]
+        action_code = ACTION_TO_CODE_MAP[action.value]
+        callback_data_args = [self.alert_group.pk, action_code]
         if action_data is not None:
             callback_data_args.append(action_data)
-        # Add org id with 'oncall-uuid' prefix to callback data.
-        # It's a workaroung to pass org_id to the oncall-gateway while proxying requests.
-        # TODO: switch to json str instead of ':' separated string.
+        # Add org id with 'oncall' prefix to callback data.
+        # It's a workaroung to pass oncall_-uuid to the oncall-gateway while proxying requests.
```
Comment (joeyorlando) [inline engine/apps/telegram/renderers/keyboard.py:107]:
```suggestion
        # It's a workaround to pass oncall-uuid to the oncall-gateway while proxying requests.
```

File: engine/apps/telegram/tests/test_keyboard_renderer.py [hunk 4]
```diff
@@ -159,19 +159,19 @@ def test_actions_keyboard_silenced(
...
-                callback_data=f"{alert_group.pk}:resolve:oncall-uuid{organization.uuid}",
+                callback_data=f"{alert_group.pk}:2:oncall-uuid{organization.uuid}",
             )
         ],
         [
             InlineKeyboardButton(
                 text="Unsilence",
-                callback_data=f"{alert_group.pk}:unsilence:oncall-uuid{organization.uuid}",
+                callback_data=f"{alert_group.pk}:5:oncall-uuid{organization.uuid}",
             )
```
Comment (joeyorlando) [inline engine/apps/telegram/tests/test_keyboard_renderer.py:175]: does it make sense to keep the tests with the action name inside of the `callback_data`, just to ensure that we still support both cases?

**Review by joeyorlando — APPROVED** [review]

**Pull request merged**